src/linear_algebra/linear_independent.lean

```diff
@@ -146,6 +146,10 @@ theorem fintype.linear_independent_iff' [fintype ι] :
     (linear_map.lsum R (λ i : ι, R) ℕ (λ i, linear_map.id.smul_right (v i))).ker = ⊥ :=
 by simp [fintype.linear_independent_iff, linear_map.ker_eq_bot', funext_iff]
 
+lemma fintype.not_linear_independent_iff [fintype ι] :
```

Comment:
The corresponding theorem on line 125 is called `linear_dependent_iff`, it would be good to keep these names consistent.

Reply:
looking at the statement `not_linear_independent_iff` seems a better name. Let me see if it's easy to change line 125.

Reply:
Yes I do agree, I would also prefer that they say not linear independent I think, but have no idea if that would be a big change to make and distract from this PR

Reply:
It seems a quite small change.